ena-submission/test/text_project_request.xml

```diff
@@ -1,7 +1,7 @@
 <PROJECT_SET>
     <PROJECT center_name="Test institution" alias="1:Test organism:Test suffix">
-        <NAME>Test scientific name</NAME>
-        <TITLE>Test scientific name: Genome sequencing</TITLE>
+        <NAME>Test scientific name: Genome sequencing by Test institution</NAME>
+        <TITLE>Test scientific name: Genome sequencing by Test institution</TITLE>
```

Comment:
Could we actually put different things in here? Is that where the subtitles come from?

Like here:
<img width="549" alt="image" src="https://github.com/user-attachments/assets/4e91dcec-5b13-48f2-811a-9419d8687a98">


It looks like it! We could put different information there then to make it look nicer

<img width="411" alt="Brave Browser 2024-10-11 12 52 24" src="https://github.com/user-attachments/assets/8367b003-468a-4b42-a06f-ac0c8500f9f5">

Reply:
I merged your changes :-)